libsrc/serialization.cpp

```diff
@@ -58,6 +58,33 @@ void serialize(const std::string &str, WritableStreamView &ws) {
     }
 }
 
+template <typename T> 
+void serialize(const NDArray<T> &arr, WritableStreamView &ws) {
+    uint16_t ver = arr.getVersion();
```

Comment:
What is the purpose of this version?

Reply:
it is in the NDArray definition of ismrmrd.